templates/patterns/list_item/pattern-list-item--variant-highlight.html.twig

```diff
@@ -10,6 +10,6 @@
       label: title,
       path: url
     },
-    image: image.src
+    image: image
```

Comment:
This seems to break the following:

![image](https://user-images.githubusercontent.com/153362/64966300-cef36200-d89e-11e9-817b-95feb12c51c4.png)

The test did not get that I guess we need to add an assertion for the image too.

Reply:
I will check again

Reply:
Are you sure that you updated the component?
Because of if `src` is empty, you don't see all div.
```
    {% if _card.image.src %}
      <div class="ecl-card__image" alt="{{ _card.image.alt }}" style="background-image:url('{{ _card.image.src }}')"></div>
    {% endif %}
```
On my local:
![image](https://user-images.githubusercontent.com/186827/64972208-cd2a9e00-d8a0-11e9-89e0-ef6e2f571730.png)

We could add an assertion for the src of the image, maybe it's a duplicate of 
`'article.ecl-card header.ecl-card__header div.ecl-card__image[alt="Alternate text for secondary image"]': 1`, after to know that the existing of the ` {% if _card.image.src %}`
As you wish.

Reply:
Yeah let's add an assertion, I'll rebuild but normally I remove all node related code before every review.

Reply:
It's all good, just me being dumb.

Reply:
I added the assertion.

Reply:
Resolved.